### DISCUSSION TITLE
Consulta respecto del requerimiento "ABOGABOT".

### DISCUSSION THREAD

**Discussion by Sebastian-Longhitano** [Launch-X-Latam/MisionFrontEnd#201] [Q&A]
Buenas noches, ¿cómo están?

Tengo una consulta respecto de la fecha en la cuál hay que tener listo a dicho requerimiento.
Entiendo que no se ha dado una fecha en específico todavía, ¿es correcto?.

Muchísimas gracias a todos los presentes!
Saludos cordiales,

**Comment by doguedogue** [comment]
asi es no han dicho, posiblemente ahorita en el live digan 🚀

**Comment by Sebastian-Longhitano** [comment]
Gracias DogueDogue!!!
